wiki/osu!_rank_one_history/en.md

```diff
@@ -0,0 +1,265 @@
+| ![][flag_TW] [Uan](https://osu.ppy.sh/users/147623) | 1 |
+| ![][flag_DE] [Whitecat](https://osu.ppy.sh/users/4504101) | 1 |
+
+## Longest Combined Reign Length
+
+*Note: Only reigns after 2011-04-23 are included.*
+
+| Player | Length (Days) |
+| --: | :-: |
+| ![][flag_KR] [chocomint](https://osu.ppy.sh/users/124493) | 926 |
+| ![][flag_TW] [hvick225](https://osu.ppy.sh/users/50265) | 579 |
+| ![][flag_PL] [WubWoofWolf](https://osu.ppy.sh/users/39828) | 467 |
+| ![][flag_DE] [Whitecat](https://osu.ppy.sh/users/4504101) | >455 |
+| ![][flag_PL] [Rafis](https://osu.ppy.sh/users/2558286) | 242 |
+| ![][flag_CL] [Mathi](https://osu.ppy.sh/users/5339515) | 212 |
+| ![][flag_JP] [rrtyui](https://osu.ppy.sh/users/352328) | 144 |
+| ![][flag_US] [idke](https://osu.ppy.sh/users/4650315) | 132 |
+| ![][flag_US] [Vaxei](https://osu.ppy.sh/users/4787150) | 77 |
+| ![][flag_KR] [Reimu-Desu](https://osu.ppy.sh/users/948713) | 70 |
+| ![][flag_KR] [FlyingTuna](https://osu.ppy.sh/users/9224078) | 59 |
+| ![][flag_KR] [sayonara-bye](https://osu.ppy.sh/users/713266) | 58 |
+| ![][flag_JP] [Flute](https://osu.ppy.sh/users/211278) | 38 |
+| ![][flag_KR] [firebat92](https://osu.ppy.sh/users/1777162) | 33 |
+| ![][flag_DE] [ShadowSoul](https://osu.ppy.sh/users/494970) | 20 |
+| ![][flag_US] [Freddie Benson](https://osu.ppy.sh/users/7342622) | 19 |
+| ![][flag_TW] [Uan](https://osu.ppy.sh/users/147623) | 11 |
+| ![][flag_FI] [thelewa](https://osu.ppy.sh/users/475021) | 1 |
+| ![][flag_US] [geckogates](https://osu.ppy.sh/users/252524) | <1 |
+
+## Longest Individual Reigns
+
+*Note: Only reigns over 100 days after 2011-04-23 are included.*
+
+| Player | Reign Length (Days) |
+| --: | :-: |
+| ![][flag_KR] [chocomint](https://osu.ppy.sh/users/124493) | 460 |
+| ![][flag_DE] [Whitecat](https://osu.ppy.sh/users/4504101) | >455 |
+| ![][flag_KR] [chocomint](https://osu.ppy.sh/users/124493) | 256 |
+| ![][flag_TW] [hvick225](https://osu.ppy.sh/users/50265) | 242 |
+| ![][flag_PL] [WubWoofWolf](https://osu.ppy.sh/users/39828) | 224 |
+| ![][flag_TW] [hvick225](https://osu.ppy.sh/users/50265) | 173 |
+| ![][flag_PL] [WubWoofWolf](https://osu.ppy.sh/users/39828) | 167 |
+| ![][flag_TW] [hvick225](https://osu.ppy.sh/users/50265) | 145 |
+| ![][flag_JP] [rrtyui](https://osu.ppy.sh/users/352328) | 144 |
+| ![][flag_KR] [chocomint](https://osu.ppy.sh/users/124493) | 142 |
+
+<!-- Includes -->
+[flag_AU]: /wiki/shared/flag/AU.gif "Australia"
+[flag_UY]: /wiki/shared/flag/UY.gif "Uruguay"
+[flag_GB]: /wiki/shared/flag/GB.gif "United Kingdom"
+[flag_CN]: /wiki/shared/flag/CN.gif "China"
+[flag_FI]: /wiki/shared/flag/FI.gif "Finland"
+[flag_JP]: /wiki/shared/flag/JP.gif "Japan"
+[flag_TW]: /wiki/shared/flag/TW.gif "Taiwan"
+[flag_PL]: /wiki/shared/flag/PL.gif "Poland"
+[flag_CL]: /wiki/shared/flag/CL.gif "Chile"
+[flag_KR]: /wiki/shared/flag/KR.gif "South Korea"
+[flag_US]: /wiki/shared/flag/US.gif "United States of America"
```

Comment:
should be `"United States"` instead of `"United States of America"` following osu-wiki's [flag references list](http://github.com/ppy/osu-wiki/blob/master/meta/flag-references/en.md)

also this should be ordered alphabetically as well i suppose (AU --> CN --> CL --> DE -->GB --> FI --> JP --> KR --> PL --> TW --> US --> UY) instead of being scattered all over

Reply:
I don't know how I missed the flag reference document, whoops.

I originally added the flags from newer->old players, but yeah, it makes sense to have them in alphabetical order.